src/Layout/topbar.tsx

```diff
@@ -38,30 +39,45 @@ const theme = createMuiTheme({
 
     }
 });
+
+export const adminLoggedIn = () => { 
+    return sessionStorage.length != 0; 
+}
+
 export const TopBar: React.FC = () => {
     const classes = useStyles();
+    const handleLogout = () => {
+        postLogout();
+    }
     return (
         <div className={classes.root}>
             <MuiThemeProvider theme={theme}>
                 <AppBar position="static">
-                    <Toolbar>
-                        <IconButton edge="start" className={classes.menuButton} color="inherit" aria-label="menu">
-                            <MenuIcon/>
-                        </IconButton>
-                        <Button color="inherit" startIcon={<DirectionsBikeIcon/>} className={classes.title}
-                                component={Link} to="/bikes">
-                            Bicycles
-                        </Button>
-                        <Button color="inherit" startIcon={<LocalParkingIcon/>} className={classes.title}
-                                component={Link} to="/stations"> Stations </Button>
-                        <Button color="inherit" startIcon={<ErrorOutlineIcon/>}
-                                className={classes.title}> Malfunctions </Button>
-                        <Button color="inherit" startIcon={<PersonOutlineIcon/>} component={Link} to="" 
-                                className={classes.title}> Login </Button>
-                    </Toolbar>
+                    {adminLoggedIn() ?
```

Comment:
Nie wiem czy branie całego Topbara w wyrażenie warunkowe to dobry pomysł, jakoś bardziej podobałoby mi się jakby Buttony coś takiego miały, żeby opcja login i logout była w tym samym miejscu na Toolbarze. To tylko moje zdanie

Reply:
Mi też się nie podoba, ze cały Toolbar jest w tym warunku. Chciałem żeby były same Buttony ale w tym wyrażeniu warunkowym może być tylko pojedynczy element więc jak chciałem żebym w jednym przypadku zwróciło trzy buttony to musiałbym je złapać jakimś jednym elementem więc wybrałem Toolbar. Jak wiesz jak to zrobić dobrze to mogę to zmienić. Z login/logout chciałem tak zrobić jak mówisz, żeby przycisk do logowania był w tym samym miejscu co wylogowanie ale nie miałem ochoty męczyć się z tym.

Reply:
Zrobiłem teraz tak, że przyciski Login i Logout są w tym samym miejscu.